browser/main/SideNav/index.js

```diff
@@ -438,9 +479,24 @@ class SideNav extends React.Component {
       )
     }
 
+    const isFolded = config.isSideNavFolded
     const style = {}
     if (!isFolded) style.width = this.props.width
     const isTagActive = location.pathname.match(/tag/)
+
+    const navSearch = (
+      <div styleName='search' style={{maxHeight: showSearch ? '3em' : '0'}}>
+        <input
+          styleName='search-input'
+          type='text'
+          ref={dom => { this.searchInput = dom }}
```

Comment:
I don't see where this `ref` is used. Please remove it if you don't use it. :smiley: